pkg/proxy/proxy.go

```diff
@@ -254,6 +254,18 @@ func findProxyPort(name string) *ProxyPort {
 	return nil
 }
 
+// AckProxyPort() marks the proxy at 'pp' as successfully created and creates or updates the datapath rules
+// accordingly.
+func (p *Proxy) AckProxyPort(name string) error {
```

Comment:
nit: The one caller always does `GetProxyPort()` prior to this call to fetch the filter name, then we end up locking & looking up `pp` in that function, then returning, and then calling into here. Did you consider having the parameters here match `GetProxyPort()` to hide that complexity from the caller?

Reply:
I did not, but that is a good point, call site becomes trivial :-) Done.